core/templates/dev/head/pages/Base.ts

```diff
@@ -26,22 +27,27 @@ require('app.constants.ts');
  */
 
 angular.module('oppia').controller('Base', [
-  '$document', '$rootScope', '$scope', 'CsrfTokenService',
-  'DocumentAttributeCustomizationService', 'MetaTagCustomizationService',
-  'SidebarStatusService', 'UrlInterpolationService', 'UrlService', 'DEV_MODE',
-  'SITE_NAME',
+  '$document', '$rootScope', '$scope', 'AlertsService', 'BackgroundMaskService',
```

Comment:
Do we have to import `BackgroundMaskService.ts` as well?

Reply:
Yes, done.
Thanks.